libs/designsystem/page/src/page.component.ts

```diff
@@ -568,6 +584,19 @@ export class PageComponent
     };
   }
 
+  private patchIonLastFocussed() {
+    const ionLastFocus: HTMLElement =
+      this.ionContentElement.nativeElement.querySelector('[ion-last-focus="true"]');
+
+    // When a focusManagerPriority is set, Ionic handles moving focus back to the element that presented the current view.
+    // Unfortunately, we need to schedule a removal of tabindex="-1" from ionLastFocus element after Ionic might have set it
+    // to work around elements becoming inaccessible after navigating back: https://github.com/ionic-team/ionic-framework/issues/29875
+    setTimeout(() => {
+      if (ionLastFocus?.getAttribute('tabindex') !== '-1') return;
+      this.renderer.removeAttribute(ionLastFocus, 'tabIndex');
```

Comment:
Is the case difference intended?
```suggestion
      this.renderer.removeAttribute(ionLastFocus, 'tabindex');
```